docs/content/2.guide/16.migrating.md

```diff
@@ -83,6 +83,38 @@ Deprecated for the same reason as the `localeLocation()` Option API.
 
 Use `localeHead()` instead for the Options API style.
 
+### Deprecated `baseUrl` component option
+
+Use `site.url` instead of `i18n.baseUrl`.
+
+::alert{type="info"}
+The `site` configuration option stems from [Nuxt SEO's `site-config` module](https://nuxtseo.com/site-config), which `nuxt-i18n` makes use of now.
```

Comment:
I think there will need to be some documentation on all the ways that site config can be set and hosted on this page. The docs aren't that friendly at the moment, something on my to-do list.